src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -0,0 +1,189 @@
+			
+		SAS UUT approves the request and responds,
+		with correct content and format for both dump message and files 
+          """
+        # STEP 1
+        # register devices
+        device_a = json.load(
+            open(os.path.join('testcases', 'testdata', 'device_a.json')))
+        device_c = json.load(
+            open(os.path.join('testcases', 'testdata', 'device_c.json')))
+        cbsd_ids = self.assertRegistered([device_a, device_c])
+        # Create grant requests
+        grant_a = json.load(
+            open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+        grant_c = json.load(
+            open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+        grant_a['cbsdId'] = cbsd_ids[0]
+        grant_c['cbsdId'] = cbsd_ids[1]
+        # Set Grants 
+        grant_a['operationParam']['operationFrequencyRange'] = {
+            'lowFrequency': 3620000000.0,
+            'highFrequency': 3630000000.0
+        }
+        grant_c['operationParam']['operationFrequencyRange'] = {
+            'lowFrequency': 3640000000.0,
+            'highFrequency': 3650000000.0
+        }
+        grant_request = [grant_a, grant_c]
+        request = {'grantRequest': grant_request}     
+        # Send grant requests
+        grant_response = self._sas.Grant(request)['grantResponse']
+        grant_ids = []
+        for resp in grant_response:
+            self.assertEqual(resp['response']['responseCode'], 0)
+            grant_ids.append(resp['grantId'])
+        # STEP 2
+        # Inject PPA
+        pal_low_frequency = 3550000000.0
+        pal_high_frequency = 3560000000.0
+        pal_record = json.load(
+          open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+        ppa_record = json.load(
+          open(os.path.join('testcases', 'testdata', 'ppa_record_0.json')))
+        ppa_record, pal_record = makePpaAndPalRecordsConsistent(ppa_record,
+                                                                [pal_record],
+                                                                pal_low_frequency,
+                                                                pal_high_frequency,
+                                                                device_a['userId'])
+        self._sas_admin.InjectPalDatabaseRecord(pal_record[0])
+        self._sas_admin.InjectZoneData({'record': ppa_record})
+        # Inject Esc Sensor
+        esc_sensor_record = json.load(
+          open(os.path.join('testcases', 'testdata', 'esc_sensor_record_0.json')))
+        self._sas_admin.InjectEscSensorDataRecord({'record': esc_sensor_record})
+        # STEP 3
+        self._sas_admin.TriggerFullActivityDump()
+        # STEP 4
+        response = self._sas.GetFullActivityDump()
+        # STEP 5
+        self.assertContainsRequiredFields("FullActivityDump.schema.json", response)
+        # STEP 6 AND CHECK
+        self.assertGreaterEqual(3, len(response['files']))
```

Comment:
Are these checks needed? Line 147 would fail if the test is not configured to inject a PPA (the current configuration injects one but the eventual configurable version of this test case will not require that). Line 148 would fail for the same reason. Moreover, what certainty do you gain by doing these checks?